certified-connectors/HiveCPQ - Product configurator/apiDefinition.swagger.json

```diff
@@ -4976,7 +5202,10 @@
             "name": "companyBId",
             "x-ms-summary": "Company ID",
             "description": "Filter on the distributor company the project segments are created for.",
-            "type": "string"
```

Comment:
These input fields "name", "type", "companyBId" have been changed from string to array. can you please confirm that no existing flows or users will be affected by this change?

Reply:
This change has been live on our internal, custom version of this connector for some time where no existing flows have seen failures because of this.

When certification proceeds we also have flows in a Power Platform Preview environment to test for any breaking issues.